src/settings.ts

```diff
@@ -765,3 +765,14 @@ export const EVENT_STARTING_SOON_TIMEFRAME = 120; // in minutes
 export const EVENTS_PREVIEW_LIST_LENGTH = 5;
 
 export const BM_PARENT_ID = "playa";
+
+// NOTE: volume numbers are expressed in %, thus between 0 and 100
+export const DEFAULT_AUDIO_VOLUME: number = 10;
+export const DEFAULT_AMBIENT_VOLUME: number = 10;
+export const DEFAULT_INTERACTIONS_VOLUME: number = 50;
+export const DEFAULT_NOTIFICATIONS_VOLUME: number = 100;
```

Comment:
looks like these constants are not used anywhere. do you think we should still leave there? the developer who is going to implement volume sliders might notice them and will create new ones.

Reply:
> looks like these constants are not used anywhere. do you think we should still leave there? the developer who is going to implement volume sliders might notice them and will create new ones.

I don't know what "might notice them and will create new ones" is, maybe a typo, but, as even from Yurii's code before, there are few things left for future expansion (alongside with sparse comments where they are used.

Reply:
I will leave them as they are since there are few more places where stubs for volume controls are put for future addition/expansion